frontend/src/app/modules/shared/components/side-bar/side-bar.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="sidebar-container">
+  <div class="sidebar-icon-container" [routerLink]="'website'" (click)="setActiveModule(DashBoardModules.Website)">
```

Comment:
Add the missing email functionalities like delete, publish create like in website.

This raises the question of handling all those in entityy so that only url changes??